bgpd/bgp_flowspec.c

```diff
@@ -108,13 +108,6 @@ int bgp_nlri_parse_flowspec(struct peer *peer, struct attr *attr,
 		return BGP_NLRI_PARSE_ERROR_FLOWSPEC_IPV6_NOT_SUPPORTED;
 	}
 
-	if (packet->length >= FLOWSPEC_NLRI_SIZELIMIT) {
```

Comment:
this should be done as part of regular bgp processing ( I think in bgp_io.c, length is checked)

Reply:
It's not checked there. That code checks that the message size is within the BGP message limits. NLRI checks are different and done in bgp_packet.c IIRC. I am asking the question because I want to make sure we are not introducing a security issue by removing a bounds check here.

Reply:
I was talking about https://github.com/FRRouting/frr/blob/master/bgpd/bgp_io.c#L226
this said, there is a limit in rfc:
'
   (0xfnnn).  The highest value that can be represented with this
   encoding is 4095.  The value 241 is encoded as 0xf0f1.
'
so one can add a check instead.